### ISSUE TITLE
Need version 1.5 so we can use `with`

### ISSUE THREAD

**Issue by wooparadog** [verigak/progress#63]
Hi,

I've found that codes(1.4) downloaded from pypi (with pip install) are different from those in repo.

To be specific, it's lacking `with` support. 

```bash
$ virtualenv test_env
$ source test_env/bin/activate
$ pip install -U progress
$ cat test_env/lib/python3.7/site-packages/progress/__init__.py | grep enter
```
A version bump should solve that.

**Comment by pquentin** [comment]
Now that #61 got accepted (thanks for the review and the merge!), I'd be happy with a new release too. 😄 But if you have other priorities, that's fine too!

**Comment by jeking3** [comment]
A release allowing use in a context manager would be great...

**Comment by verigak** [comment]
Released v1.5

**Comment by jeking3** [comment]
Already using it - thanks!
